font-icons/controls-icons.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Controls Icons
+page_title: .NET MAUI Controls Icons - Overview
+description: Telerik font icons are collections of small vector graphics used across the components in the Telerik UI for .NET MAUI suite.
+slug: telerik-controls-icons
+position: 2
+---
+
+<link rel="stylesheet" href="style-controls.css" />
+
+# Telerik UI for .NET MAUI Controls Icons
+
+You can choose any of the available Telerik Font icons: 
```

Comment:
In addition to the suggested change, think about expanding the intro a bit. What are Controls Icons, where would people use them or where are they visible in action?

```suggestion
You can choose any of the available Telerik font icons: 
```